Makefile

```diff
@@ -42,17 +41,17 @@ check:
      done; \
      exit "$$EXIT_CODE";
 
-dockersetup:
```

Comment:
Don't remove this, it will break everything when used on a system which hasn't pulled the docker images before.